docs/source/index.rst

```diff
@@ -382,6 +382,8 @@ Flax), PyTorch, and/or TensorFlow.
 +-----------------------------+----------------+----------------+-----------------+--------------------+--------------+
 |       Encoder decoder       |       ❌       |       ❌       |       ✅        |         ❌         |      ✅      |
 +-----------------------------+----------------+----------------+-----------------+--------------------+--------------+
+|             ESM             |       ✅       |       ✅       |       ✅        |         ❌         |      ❌      |
```

Comment:
Please also add your model in the main README (in alphabetical order) then run `make fix-copies` on your branch.